CHANGELOG.md

```diff
@@ -3,7 +3,33 @@
 ## Unreleased
 
 ### Breaking changes
-You must make the following changes when you migrate to this release, or your service may break.
+Where applicable, you must make the following changes when you migrate to this release, or your service may break.
+
+#### Check your accordions still display as expected
+
+[See the new design and revised guidance for accordions in the Design System](https://design-system.service.gov.uk/components/accordion/). In particular, review the guidance on when and when not to use accordions.
+
+You should check that your accordions still display as expected if you:
+
+- have applied any custom styles to your accordions
+- use accordions in non-standard ways
+- have section headings that contain unusual formatting (for example, wrappers)
+
+You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
+
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
+
+#### Keep summary line content brief
```

Comment:
Just a quick one - could this be
```
Keep the accordion summary line content brief
```

to make it clear to anyone scanning this that it's about the accordion.

Reply:
@hannalaakso That sounds like a good idea. Just thought of something else as well - the bit of content that follows is also about users checking whether they need the summary line. Wonder do we need to cover that in the heading too?

```suggestion
#### Review and revise your summary line
```

Reply:
@EoinShaughnessy That's up to you. I kind of think that people might still expect to see the calls to action inside the content and for the heading to summarise what's in the content. But that's more of a content design question so I'll leave that to you.

But I like your proposed heading
> Review and revise the accordion summary line content

and we could just keep the content as it is.

Reply:
Thanks @hannalaakso, that sounds like a neat idea.

```suggestion
#### Review and revise the accordion's summary line content
```